torch/distributed/fsdp/_debug_utils.py

```diff
@@ -53,7 +53,9 @@ def profile(cls, profile_type: str) -> Iterator[None]:
 
     @classmethod
     def dump_and_reset(cls, msg: str) -> None:
-        logger.warning("%s %s", msg, str(cls.results))
+        # This cannot be combined with DETAIL distributed log
+        # as the profiling will be very incorrect.
+        logger.debug("%s %s", msg, str(cls.results))
```

Comment:
```suggestion
        logger.debug("%s %s", msg, cls.results)
```
should work as well and be more efficient